app/views/content/teacher-training-advisers.md

```diff
@@ -103,15 +103,11 @@ $ql-2$
 
 ## Can I get an adviser?
 
-To sign up, you’ll need (one of the following):
+To sign up, you’ll need to have or be in your final year of studying for a bachelor's degree with honours, grade 2:2 or higher (or an equivalent).
```

Comment:
@EllieNodder I find this sentence a bit confusing now it is mushed together, particularly when grade is thrown in there as well